docs/src/add-services/varnish.md

```diff
@@ -146,8 +144,12 @@ This limitation may be lifted in the future.
 
 ## Stats endpoint
 
-The Varnish service also offers an `http+stats` endpoint, which provides access to some Varnish analysis and debugging tools.
-To access it, from a dedicated app container add the following to your [app configuration](../create-apps/app-reference.md):
+The Varnish service also offers an `http+stats` endpoint,
+which provides access to some Varnish analysis and debugging tools.
+
+You can't use it from an app fronted by Varnish because of the restriction with [circular relationships](#circular-relationships).
+To access the stats, create a **separate app** with a relationship *to* Varnish, but not *from* it.
```

Comment:
:question:
Wouldn't it require a change in the `.platform/routes.yaml` as well?

Reply:
If you wanted to make the app available to the outside world, I guess so. Right now the flow is just using SSH, so it doesn't seem necessary. It's up to them how they'd like to do it, though. It's not an ideal workaround, so I'm not sure I'd spend too long polishing it rather than just clearing up the current bug.

Reply:
The following:
> When the app is deployed, the app can access the Varnish service over HTTP to get diagnostic information.
Got me confused.

So that part of the flow could be a bit clearer, but that's maybe something for the bigger rework then.

Reply:
Will make a note to come back to it. It's not ideal.